cypress/support/table-commands.ts

```diff
@@ -39,6 +39,7 @@ Cypress.Commands.add('selectTableFilter', (dataCy: string) => {
   // We need to use document() to get the body of the page
   // and then find the filter-select within that body
   // This fixes the issue where this command would fail inside a within() block
+  cy.document().its('body').find('#filter-search').type(dataCy.replaceAll('-', ' '));
```

Comment:
Remove - this isn't being called anywhere in the tests.

Reply:
The `selectTableFilter` command is called in the `filterTableByTextFilter` command which is used here https://github.com/ansible/ansible-ui/pull/1943/files#diff-70b1ba24eeb1e57585230c8e46e8acf6752d0316bc172f916da27340543eff49R52. This change was necessary in order for the filter to be visible in cypress.